tests/fuzzing/oss_fuzz_build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+mv $SRC/id_map_fuzzer.go $SRC/runc/libcontainer/system/
```

Comment:
1. Who calls this script (and sets `$SRC`)?

Reply:
The script is only meant to be run by OSS-fuzz who also sets `$SRC`. I have added a bit of documentation inside the build file itself.